src/bindings/python/tests/test_runtime/test_compiled_model.py

```diff
@@ -276,3 +276,21 @@ def test_compiled_model_from_buffer_in_memory(request, tmp_path, device):
 
     compiled = core.compile_model(model=xml, weights=weights, device_name=device)
     _ = compiled([np.random.normal(size=list(input.shape)).astype(dtype=input.get_element_type().to_dtype()) for input in compiled.inputs])
+
+
+def test_memory_release():
+    core = Core()
+
+    input_shape = [5]
+
+    params = [ops.parameter(input_shape, np.float32), ops.parameter(input_shape, np.float32)]
+
+    model = Model(ops.concat(params, 0), params)
```

Comment:
do we have somewhere predefined test model in utils? instead of creating each time in each test

Reply:
Moved it to utils